hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -755,46 +797,27 @@
             FSM transition.
 
             The main_pok from AST is randomly forced to flip while in the middle of a FSM
-            transition. This is done on all normal / deep sleep / reset request tests.
+            transition. This is done on all normal / deep sleep / reset request tests. The check
+            that flipping during POR results in a stretched reset is done at IP level.
             '''
       milestone: V2
       tests: []
     }
     {
       name: chip_pwrmgr_b2b_sleep_reset_req
-      desc: '''Verify that the pwrmge sequences sleep_req and reset req coming in almost at the same
+      desc: '''Verify that the pwrmgr sequences sleep_req and reset req coming in almost at the same
             time, one after the other.
 
+            Verified at IP level.
             '''
       milestone: V2
       tests: []
     }
     {
       name: chip_pwrmgr_debug_sleep
       desc: '''Verify that when the chip being in "debuggable" state prevent the low power entry.
-            '''
-      milestone: V2
-      tests: []
-    }
-    {
-      name: chip_pwrmgr_sleep_wake_req_disabled
-      desc: '''Verify that the chip cannot be woken up from sleep from a wake up source that is
-            disabled.
-            '''
-      milestone: V2
-      tests: []
-    }
-    {
-      name: chip_pwrmgr_reset_req_disabled
-      desc: '''Verify that the chip cannot be reset from a reset source that is disabled.
-            '''
-      milestone: V2
-      tests: []
-    }
-    {
-      name: chip_pwrmgr_sleep_reset_req_disabled
-      desc: '''Verify that the chip cannot be woken up from sleep from a reset source that is
-            disabled.
+
+            This is an open issue.
```

Comment:
I think we should attach `TODO(#bug)` items to track this a bit more easily.

Reply:
I found this that is related https://github.com/lowRISC/opentitan/issues/7215.